wsit/ws-sx/wssx-impl/src/main/java/com/sun/xml/wss/impl/resolver/ResolverId.java

```diff
@@ -357,5 +356,13 @@ private void nodeSetMinusCommentNodes(Node node, Set nodeSet,
 		nodeSet.add(node);
 	}
    }
+
+    public XMLSignatureInput engineResolveURI(ResourceResolverContext context) throws ResourceResolverException {
+        return null;
```

Comment:
it is always needed to nullify the result?

Reply:
Agreed, should we not mirror upstream?

```suggestion
        return engineResolve(rrc.attr, rrc.baseUri);
```